mne/realtime/lsl_client.py

```diff
@@ -0,0 +1,119 @@
+# Authors: Teon Brooks <teon.brooks@gmail.com>
+#          Mainak Jas <mainak@neuro.hut.fi>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+from .base_client import _BaseClient
+from ..epochs import EpochsArray
+from ..io.meas_info import create_info
+from ..io.pick import _picks_to_idx, pick_info
+from ..utils import fill_doc, logger
+
+try:
+    import pylsls
```

Comment:
wouldn't this break importing of MNE if pylsls was not present?

Reply:
i just found the common pattern for doing some within mne. replacing with such